sce.sh

```diff
@@ -122,16 +141,19 @@ then
 fi
 
 name=""
+configPath=""
 is_quasar_alias "$2"
 if [ $? -eq 0 ]
 then
     name=$QUASAR_REPO_NAME
+    configPath="config/config.json"
 fi
 
 is_clark_alias "$2"
 if [ $? -eq 0 ]
 then
     name=$CLARK_REPO_NAME
+    configPath="src/config/config.json"
```

Comment:
there is also `api/config/config.json` in clark, i forgot about this

Reply:
done